docs/Reference.md

```diff
@@ -318,7 +318,7 @@ Discord uses ids and hashes to render images in the client. These hashes can be
 | Guild Discovery Splash      | discovery-splashes/[guild_id](#DOCS_RESOURCES_GUILD/guild-object)/[guild_discovery_splash](#DOCS_RESOURCES_GUILD/guild-object).png                                                                                                                                | PNG, JPEG, WebP      |
 | Guild Banner                | banners/[guild_id](#DOCS_RESOURCES_GUILD/guild-object)/[guild_banner](#DOCS_RESOURCES_GUILD/guild-object).png \*                                                                                                                                                  | PNG, JPEG, WebP, GIF |
 | User Banner                 | banners/[user_id](#DOCS_RESOURCES_USER/user-object)/[user_banner](#DOCS_RESOURCES_USER/user-object).png \*                                                                                                                                                        | PNG, JPEG, WebP, GIF |
-| Default User Avatar         | embed/avatars/[user_discriminator](#DOCS_RESOURCES_USER/user-object).png \*\* \*\*\*                                                                                                                                                                              | PNG                  |
+| Default User Avatar         | embed/avatars/[user_id](#DOCS_RESOURCES_USER/user-object).png \*\* \*\*\*                                                                                                                                                                              | PNG                  |
```

Comment:
```suggestion
| Default User Avatar         | embed/avatars/[user_created_timestamp](#DOCS_RESOURCES_USER/user-object).png \*\* \*\*\*                                                                                                                                                                              | PNG                  |
```
`user_id % 5` and `(user >> 22) % 5` can not be the same

Reply:
The API would do that conversion though wouldn't it?

Reply:
This is not an API route, but CDN

Reply:
user_id is correct
with user_discriminator the calculation was `user_discriminator % 5` as well

Reply:
The problem here is that this is listed as `user_id` even though it is computed to _be_ `(user_id >> 22) % 5` but it is listed away as a note for some reason, and that reason is very likely "I couldn't find a better name for it." Just wish it was in here instead as at a glance it is inaccurate unless you read its disclaimer.

Reply:
Fair. Maybe a new section could be added for this docs part and be linked in this table like calculated_hash

And in the section the calculation method then.